### PR TITLE
Bugfix/dhfprod 653

### PR DESCRIPTION
**Pull request by grechaw**
Fixes issues with tracing on clustered environments.

### CODE REVIEW

**Comment by srinathgit** [comment]
@grechaw ,

I just ran the test from one vpc against ALB which is in another vpc (along with MarkLogic). I had a bunch of tracing tests failing. They passed earlier when I ran from my local machine .

```
com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runXqyXmlFlowWithBinaryContent STARTED

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runXqyXmlFlowWithBinaryContent FAILED
    java.lang.AssertionError: expected:<4> but was:<1>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:645)
        at org.junit.Assert.assertEquals(Assert.java:631)
        at com.marklogic.hub.job.TracingTest.runXqyXmlFlowWithBinaryContent(TracingTest.java:171)

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runJSONErrorFlowWithoutTracing STARTED

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runJSONErrorFlowWithoutTracing PASSED

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runXqyJsonFlowWithBinaryContent STARTED

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runXqyJsonFlowWithBinaryContent FAILED
    java.lang.AssertionError: expected:<4> but was:<1>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:645)
        at org.junit.Assert.assertEquals(Assert.java:631)
        at com.marklogic.hub.job.TracingTest.runXqyJsonFlowWithBinaryContent(TracingTest.java:216)

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runXMLFlowWithTracing STARTED

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runXMLFlowWithTracing PASSED

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runXMLFlowSansTracing STARTED

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runXMLFlowSansTracing PASSED

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runJSONFlowWithTracing STARTED

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runJSONFlowWithTracing FAILED
    java.lang.AssertionError: expected:<6> but was:<1>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:645)
        at org.junit.Assert.assertEquals(Assert.java:631)
        at com.marklogic.hub.job.TracingTest.runJSONFlowWithTracing(TracingTest.java:243)

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runJSONFlowSansTracing STARTED

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runJSONFlowSansTracing PASSED

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runSjsXmlFlowWithBinaryContent STARTED

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runSjsXmlFlowWithBinaryContent FAILED
    java.lang.AssertionError: expected:<6> but was:<1>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:645)
        at org.junit.Assert.assertEquals(Assert.java:631)
        at com.marklogic.hub.job.TracingTest.runSjsXmlFlowWithBinaryContent(TracingTest.java:303)

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runSjsJsonFlowWithBinaryContent STARTED

com.marklogic.hub.CoreTestSuite > com.marklogic.hub.job.TracingTest.runSjsJsonFlowWithBinaryContent FAILED
    java.lang.AssertionError: expected:<6> but was:<1>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:645)
        at org.junit.Assert.assertEquals(Assert.java:631)
        at com.marklogic.hub.job.TracingTest.runSjsJsonFlowWithBinaryContent(TracingTest.java:269)

```

File: marklogic-data-hub/gradle.properties [hunk 1]
```diff
@@ -13,7 +13,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 mlDHFVersion=4.0.0
-mlHost=localhost
+mlHost=fedor
```
Comment (srinathgit) [inline marklogic-data-hub/gradle.properties]: 'mlHost' and 'mlSecurityPassword' has to be changed

**Comment by grechaw** [comment]
oh dammit i rarely check that in by mistake.

**Comment by grechaw** [comment]
There are alternate approaches, it has more to do with what works fastest and is least obtrusive for non-tracing-enabled environments.

**Pull request merged**